doc/release-process.md

```diff
@@ -56,22 +57,23 @@ In `configure.ac` and `clientversion.h`:
 
 - Change `CLIENT_VERSION_IS_RELEASE` to false while Zcash is in beta-test phase.
 
-If this release changes the behavior of the protocol or fixes a serious bug, we may
-also wish to change the `PROTOCOL_VERSION` in `version.h`.
-
```

Comment:
I didn't change it.  I think this PR just needs to be rebased to include the commit which introduced that text:
https://github.com/zcash/zcash/commit/0d317cd522ff04796b3a9ecb5ef58cca44d3920b